contracts/src/groth16/ControlID.sol

```diff
@@ -20,6 +20,7 @@
 pragma solidity ^0.8.9;
 
 library ControlID {
-    uint256 public constant CONTROL_ID_0 = 0x39ff805954f4eb2868d338764408f76d;
-    uint256 public constant CONTROL_ID_1 = 0x15cf3a5f4097269e3a6d921c18625531;
+    uint256 public constant CONTROL_ID_0 = 0x17da5b0946fcb1115de33dca38ca0d4c;
+    uint256 public constant CONTROL_ID_1 = 0x4fa8b5b0563340e93606acfe2f6ab8d5;
```

Comment:
This is a standing issue, but we need to clarify these names. In particular, `CONTROL_ID_x` is mileading because there are actually the control root / allowed program root.

Reply:
Yeah, I need to open an issue, I had a discussion w/ Paul/Frank/Jeremy choosing the name `BN254_CONTROL_ID` where we discussed also cleaning up the other related names (including these) -- but it was not on ceremony critical path so I haven't done it yet.